db/migrate/20210103181821_seed_locations.rb

```diff
@@ -0,0 +1,10 @@
+class SeedLocations < ActiveRecord::Migration[6.1]
+  def up
+    Rake::Task['db:seed_locations'].invoke
+  end
+
+  def down
+    City.delete_all
+    County.delete_all
```

Comment:
Since we're doing so in other db migrations, we could do the same here (prevent migrations failing when/if we remove these models in the codebase):

```rb
  City = Class.new(ActiveRecord::Base)
  County = Class.new(ActiveRecord::Base)
```

Reply:
👍 Forgot to add them when writing down method.